packages/components/input-number/demo/Keyboard.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <IxSpace>
+    <IxInputNumber v-model:value="value" :keyboard="keyboard" :max="10" :min="1"></IxInputNumber>
```

Comment:
```suggestion
    <IxInputNumber v-model:value="value" keyboard :max="10" :min="1"></IxInputNumber>
```

这样就好了，boolean 类型的可以简化处理。